content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -753,15 +754,68 @@ rules:
 
 See [Authorization Overview](/docs/reference/access-authn-authz/authorization/).
 
-## Server Side Apply
+## Updates to existing resources {#patch-and-apply}
+
+You can overwrite (**update**) an existing resource - for example, a ConfigMap -
+using an HTTP PUT. For a PUT request, it is the client's responsibility to specify
+the `resourceVersion` (taking this from the object being updated). Kubernetes uses
+that `resourceVersion` information so that the API server can detect lost updates
+and reject requests made by a client that is out of date with the cluster.
+In the event that the resource has changed (the `resourceVersion` the clienti
+provided is stale), the API server returns a `409 Conflict` error response.
+
+Instead of sending a PUT request, the client can send an instruction to the API
+server to **patch** an existing resource. A **patch** is appropriate if the
+change that the client wants to make isn't conditional on the existing data.
+Clients that need effective detection of lost updates should consider using
+HTTP PUT, and retrying in case of conflict.
+
+The Kubernetes API supports four different PATCH operations, determined by their
+corresponding HTTP `Content-Type` header:
+
+`application/apply-patch+yaml`
+: Server Side Apply YAML (a Kubernetes-specific extension, based on YAML).
+  All JSON documents are valid YAML, so you can also submit JSON using this
+  media type. See [Server Side Apply serialization](/docs/reference/using-api/server-side-apply/#serialization)
+  for more details.  
+  To Kubernetes, this is a **create** operation if the object does not exist,
+  or a **patch** operation if the object already exists.
+
+`application/json-patch+json`
+: JSON Patch, as defined in [RFC6902](https://tools.ietf.org/html/rfc6902).
```

Comment:
I don't know if it's worth mentioning that one can use the "test" operation of json-patch to guarantee some consistency in their patch.

Reply:
Could that go in a separate PR?

“guarantee” and “some” don't sit well with each other. What's the story here?

Reply:
Done (in outline). 

[I hope that] details can come in a follow-up PR by someone who understands it.

Reply:
Looks good to me thanks